docs/finalproject/external-projects.md

```diff
@@ -0,0 +1,127 @@
+
+In the ideal world, you would be able to answer questions within a 0.5-1 working days of the questions being asked by our students. If this might not be possible, let us know and we will select an Internal Product Manager who will be empowered to make decisions on your behalf.
+
+## What is a Good Project?
+
+It's important that the projects that we work on fulfill some base requirements to make sure they go smoothly.
+
+All Final Projects aim to be a [Minimum Viable Product](https://www.agilealliance.org/glossary/mvp/). This means they may be rough around the edges but have all of the core functionality working.
+
+### Functionality
+
+A Good Final Project should have the following qualities
+
+1. Be an interactive website
+
+   - This means a website that a user can interact with to create, read, upload, delete data.
+   - For example, a photo hosting service or blog would fulfil this criteria
+   - A static website like the [CodeYourFuture website](https://www.codeyourfuture.io) would **not**
+
+2. Be completable in four weeks of work
+
+   - This means the website should not be too large that they can't finish it or too small that they complete it too quickly
+   - For example, a newsletter sign-up form would be too small
+   - For example, a full content management system would be too large
+
+3. Be well-defined
+
+   - This means that from day one our students understand what they are building
+   - For example, a well defined project should be able to answer using [User Stories](https://www.mountaingoatsoftware.com/agile/user-stories) **all** of the functionality of the project
+   - We **can** work with Charities to define projects if they contact us early enough
+
+### Technical Functionality
+
+_Don't know what anything here means? That's fine! The above section covers this in less technical details_
+
+We use the following technologies at CodeYourFuture
+
+- JavaScript
+- React
+- Node with Express
+- MongoDB or PostgreSQL
+
+An external Final Project should use all of these technologies into a single Full Stack package. This is important as we want our students to use all of the skills that they have learnt in our course.
+
+We advise against using other tools as they will fall outside our students areas of competency due to the time constraints that we have.
+
+### Limitations
+
+As Junior Developers, our students do have some limits on what they can produce
+
+- Visual Design Complexity
+  - An overly complex design will take a lot longer for our students to develop. Try to keep your designs simple and functional.
+- Instant Answers
+  - You shouldn't expect our students to be able to give instant answers to any technical questions you might have. They will get there with time however!
```

Comment:
Not sure if this is the best place to mention this, but do we want to touch on volunteer support? Maybe worth covering off the idea that volunteers are there to help the students not necessarily the project itself?